state/bulk.go

```diff
@@ -16,21 +16,29 @@ package state
 import (
 	"context"
 	"encoding/json"
+	"errors"
 )
 
-// BulkGetOpts contains options for the BulkGet method
+// BulkGetOpts contains options for the BulkGet method.
 type BulkGetOpts struct {
 	// Number of requests made in parallel while retrieving values in bulk.
 	// When set to <= 0 (the default value), will fetch all requested values in bulk without limit.
 	// Note that if the component implements a native BulkGet method, this value may be ignored.
 	Parallelism int
 }
 
+// BulkStoreOpts contains options for the BulkSet and BulkDelete methods.
+type BulkStoreOpts struct {
+	// Number of requests made in parallel while storing/deleting values in bulk.
+	// When set to <= 0 (the default value), will perform all operations in parallel without limit.
+	Parallelism int
+}
+
 // BulkStore is an interface to perform bulk operations on store.
 type BulkStore interface {
```

Comment:
Does this break pluggable components?

Reply:
Actually.... It shouldn't. The [protos](https://github.com/dapr/dapr/pull/6342/files#diff-2179950bcc2e9039862417bd7c8e169b893f5a54dbdc2c62cf04f225502b6584) have changed but we've only added new properties, which should be backwards-compatible.

Btw my understanding was that pluggable components are not stable and we are allowed to break them if we need to change the APIs in contrib. Can you confirm that's the case?

Reply:
We are allowed to break but it does not mean we can leave it broken. We can validate that in the RC.